projects/validation-showcase/src/app/visual/visual.component.html

```diff
@@ -0,0 +1 @@
+<p>visual works!</p>
```

Comment:
Would you mind porting over the landing page links so developers can easily navigate to the visual tests?